### ISSUE TITLE
Env info failure

### ISSUE THREAD

**Issue by adamf321** [rvtraveller/terminus-composer#8]
It's failing with the following error:

```
Fatal error: Call to undefined method Terminus\Models\Environment::info() in /Users/adam/terminus/plugins/terminus-composer/Commands/ComposerCommand.php on line 61
```

This is the command I ran: ```terminus composer "install" --site=nystec --env=dev``` 

Terminus version 0.13.3

**Comment by greg-1-anderson** [comment]
Possibly duplicate of https://github.com/rvtraveller/terminus-composer/pull/7

**Comment by rvtraveller** [comment]
Hi @adamf321 I just merged #7 which should resolve this issue.  Let me know if that doesn't fix things for you.

**Comment by greg-1-anderson** [comment]
I am now finding that the HEAD of master for terminus composer is working with terminus 0.13.3. Thanks.
